### ISSUE TITLE
one character wide for chat window

### ISSUE THREAD

**Issue by ComputingFroggy** [FreeRDP/Remmina#755]
Hi,

I am running Bodhi 3.1 (Ubuntu 14.04 derivatives using Enlightenment 17 as window manager).
In the chat window, the area where I type in is one (and a bit) character wide, this is not really handy to read what is being typed in.

Is there a way to change that ?

Cheers,
L@u

**Comment by antenore** [comment]
I'll take a look later.

**Comment by antenore** [comment]
Which VNC server do you have? I use UltraVNC and TightVNC, but I cannot even start the chat session. It looks like is not supported/implemented.

**Comment by antenore** [comment]
True, it's ridiculous! The chat window is something I never had the time to test correctly.

Be patient, I'll have to rewrite most of it.

**Comment by antenore** [comment]
As I said I cannot test this, so I've modified the code without really knowing what will happens (well I wrote some testing code).

Can you please test it? It should be available in few hours in the PPA.

I also like to see some screenshots.

Thanks!

**Comment by ComputingFroggy** [comment]
Hi, I am running UltraVNC on the Windows machine.
I will update my Linux box and I will post screenshots.

**Comment by ComputingFroggy** [comment]
Hi,
Sorry for the delay: it's much better now.

There's still a little nag, the top area is not editable, but the cursor can be positioned inside it ... this could be confusing (particularly at the start when nothing has been written yet).

Here's a screenshot : https://framapic.org/NqNc9Vj36nBy/L5q1JArTjo0D.png

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @ComputingFroggy
